README.md

```diff
@@ -41,6 +41,22 @@ _Update path to fit your environment_
 }
 ```
 
+### For Linux / Unix
+
+If you're running OmniSharp on Linux or Unit, you need to set `DOTNET_REFERENCE_ASSEMBLIES_PATH` for projects target desktop CLR. You can use following bash script to set `DOTNET_REFERENCE_ASSEMBLIES_PATH` to mono's reference assemblies folder
```

Comment:
:) Thank you for point that out. I’ll update. This is the reason why my English teachers never liked me.